doc/new-book/managing-using-qdmanage.adoc

```diff
@@ -210,6 +211,13 @@ a|
 qdmanage query role port --type=connector
 ----
 
+|If the router is connected to a broker, view the alternate URLs on which the router can connect to the broker if the primary connection fails
+a|
+[options="nowrap",subs="+quotes"]
+----
+qdmanage query failoverList --name=_CONNECTOR_NAME_
```

Comment:
to be consistent the qdmanage query command should look like this - 

qdmanage query failoverList --type=connector --name=_CONNECTOR_NAME_

Reply:
Done.